doc/CODING.md

```diff
@@ -51,33 +51,56 @@ intent to add a new rule here.
 
 See [DESIGN](DESIGN.md) document too, they complement each other.
 
+### Coding Style ###
 
-### C Guidelines ###
+- Limits
 
  * Functions should not exceed 100 lines.
  * Files should not exceed 1000 lines.
- * A line should not be longer than 72 characters.
- * Split up when those limits are reached.
- * Rationale: Readability with split windows.
- * The compiler shall not emit any warning (or error).
+ * A line should not be longer than 80 characters.
```

Comment:
Yes, the question was directed to everyone. So ok, if you cannot live with 120, we won't use it. I can live with 80, and agree that too long lines are hard to read.

Nevertheless, one last try: What about something around 100? The rationale behind 100 it is: If we agree that 80 characters in a line are ideal to read, but we often have 2 tabs (takes 16 spaces, 2 chars), something around 100 would be ideal.

Btw. Many setups, including e-mail editors are often configured with 72, so linewraps can also occur with 80 chars.